generators/generator-base.js

```diff
@@ -230,7 +230,10 @@ module.exports = class JHipsterBaseGenerator extends PrivateBase {
     getPrettierExtensions() {
         let prettierExtensions = 'md,json,yml';
         if (!this.skipClient && !this.jhipsterConfig.skipClient) {
-            prettierExtensions = `${prettierExtensions},js,ts,tsx,vue,css,scss`;
+            prettierExtensions = `${prettierExtensions},js,ts,tsx,css,scss`;
+            if(this.clientFramework && this.clientFramework === VUE){
```

Comment:
```suggestion
            if (this.clientFramework && this.clientFramework === VUE) {
```

Reply:
I think this was on outdated commit. Formatted file in last commit

Reply:
yes, you fixed it just before I added my comment :D